plugins-base/XSFeatureNetworkReset.py

```diff
@@ -365,18 +365,23 @@ def Commit(self):
 		inventory['CURRENT_INTERFACES'] = ''
 		write_inventory(inventory)
 
+		ipv6 = self.IP.find(':') > -1
+
 		# Rewrite firstboot management.conf file, which will be picked it by xcp-networkd on restart (if used)
 		f = open(management_conf, 'w')
 		try:
 			f.write("LABEL='" + self.device + "'\n")
-			f.write("MODE='" + self.mode + "'\n")
```

Comment:
This will get hairy with dual-stack ip, it needs a rethinking when that is introduced so both ipv4 and ipv6 can b written independently. As it is, it will get difficult to read with 3 modes.